Toolkit.py

```
@@ -1,321 +1,321 @@
+		winreg.CreateKey(regkey,execPath);
+		regkey.Close();
+		regkey = winreg.OpenKey(winreg.HKEY_CURRENT_USER, f'Console\\{execPath}',reserved=0, access=winreg.KEY_WRITE);
+		winreg.SetValueEx(regkey,'CodePage',0,winreg.REG_DWORD,65001);
+		winreg.SetValueEx(regkey,'WindowSize',0,winreg.REG_DWORD,262174);
+		winreg.SetValueEx(regkey,'WindowAlpha',0,winreg.REG_DWORD,(config.ConsoleAlpha * 255 // 100));
+		winreg.SetValueEx(regkey,'FaceName',0,winreg.REG_SZ,'霞鹜文楷等宽');
+		winreg.SetValueEx(regkey,'FontSize',0,winreg.REG_DWORD,(config.ConsoleFontSize << 16));
+		regkey.Close();
+		logger.debug(f"ChangeConsoleStyle() Run Time: {(time.perf_counter_ns() - startTime)/1000000} ms");
+
+	def GetHash(filePath:str=None)->str:
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)